packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -236,19 +236,6 @@ class RunCommand extends RunCommandBase {
     };
   }
 
-  @override
-  void printNoConnectedDevices() {
-    super.printNoConnectedDevices();
-    if (getCurrentHostPlatform() == HostPlatform.darwin_x64 &&
```

Comment:
I was about to migrate this if statement but I realized it was dead code as of https://github.com/flutter/flutter/pull/35084.